waitress/app/views.py

```diff
@@ -52,4 +55,35 @@ class Dashboard(LoginRequiredMixin, View):
     login_url = "/"
 
     def get(self, request):
-        return render(request, "dashboard.html")
+        users_query_set = SlackUser.objects.order_by("id")
+        users_list = users_query_set.all()
```

Comment:
This means that you would have to have a query of page number alongside twhat is being passed here